backend/app/controllers/uoas.js

```diff
@@ -146,8 +146,20 @@ module.exports = {
             if (_.first(result)) {
                 throw new HttpError(403, 'Subject used in Subject to Tag link. Could not delete Subject');
             }
-            return yield thunkQuery(UnitOfAnalysis.delete().where(UnitOfAnalysis.id.equals(req.params.id)));
-        }).then(function () {
+
+            yield thunkQuery(
```

Comment:
I would say we do a soft delete for all these.

Reply:
PR.